assets/js/modules/analytics-4/datastore/accounts.js

```diff
@@ -73,7 +70,7 @@ const fetchGetAccountSummariesStore = createFetchStore( {
 			...state,
 			accountSummaries: [
 				...( state.accountSummaries || [] ),
-				...( response.accountSummaries || [] ),
+				...populateAccountSummaries( response.accountSummaries || [] ),
```

Comment:
This fixes a potential bug where `findMatchedProperty` could be invoked after account summaries had been received, but before they had been transformed/populated with their IDs which happened in an action once all AS's had been received. This resulted in all `propertyID`s being `undefined` and obviously caused problems with proper matching. That wasn't really necessary, as only the sorting needs to be done at the end.